doc/documentation/connection.rst

```diff
@@ -24,8 +25,8 @@ A context notation (using ``with:``) can be used to open a connection:
 .. code:: python
 
    >>> import pyads
-   >>> plc = pyads.Connection('127.0.0.1.1.1', pyads.PORT_TC3PLC1)
-   >>> with plc:
+   >>> TARGET_NET_ID = '5.12.82.20.1.1'  # OR pyads.ads.adsGetNetIdForPLC('192.168.3.9') if unknown
```

Comment:
Actually using 127.0.0.1.1.1 is on purpose as it refers to the locals AMS-ID of your local system. So I ask you to use this as a default. But bringing on the adsGetNetIdForPLC function is a nice touch.